cmd/gitops/beta/run/cmd.go

```diff
@@ -361,9 +361,14 @@ func dashboardStep(ctx context.Context, log logger.Logger, kubeClient *kube.Kube
 	case install.DashboardTypeOSS:
 		log.Successf("GitOps Dashboard is found")
 	default:
-		wantToInstallTheDashboard := false
+		const (
+			unknown = iota
+			wantToInstall
+			doNotWantToInstall
+		)
+		wantToInstallTheDashboard := unknown
 		if dashboardHashedPassword != "" {
-			wantToInstallTheDashboard = true
+			wantToInstallTheDashboard = wantToInstall
 		} else if !flags.SkipDashboardInstall && dashboardHashedPassword == "" {
```

Comment:
```suggestion
		} else if flags.SkipDashboardInstall {
			wantToInstallTheDashboard = doNotWantToInstall
		} else {
```
This makes it a little easier to read.